inginious/frontend/app.py

```diff
@@ -220,7 +222,8 @@ def get_app(config):
                                      'frontend/templates/layout_lti',
                                      config.get('use_minified_js', True))
 
-
+    is_tos_defined = fs_provider.exists(os.path.join(config.get("static_directory", "./static"), "terms.yaml")) and \
```

Comment:
As explained, I'd rather have the two pages specified in the ``configuration.yaml`` file, as this is already the case for ``welcome_page`` instead of hardcoding their names. This would make behaviour more uniform. The ``is_tos_defined`` can therefore become a check if those paramter are specified in the ``configuration.yaml``file.